README.md

```diff
@@ -140,6 +144,8 @@ Some metrics are not available with Artifactory OSS license. The exporter return
 
 | Metric | Description | Labels | OSS support |
 | ------ | ----------- | ------ | ------ |
+| artifactory_storage_repo_quota_used_percent | Percentage of quota used |`version`, `revision`, `branch`, `goversion` | ? |
+| artifactory_storage_repo_quota_bytes  | Artifactory quota in bytes |`version`, `revision`, `branch`, `goversion` | ? |
```

Comment:
The list of labels is not correct. You can confirm the list of labels by looking at the metrics we had from the running system before.

Reply:
found the labels from running the system and changed to correct labels